packages/node_modules/@ciscospark/common/src/one-flight.js

```diff
@@ -58,10 +58,6 @@ export default function oneFlight(...params) {
         if (this && this.logger) {
           this.logger.info(message);
         }
-        else {
```

Comment:
Let's do this with check on NODE_ENV and/or CISCOSPARK_LOG_LEVEL rather than removing it entirely. It is *useful*, if *annoying* :)

Reply:
I don't see the pattern of the logger being used in conjunction with NODE_ENV anywhere else. Shouldn't we just rely on the logger config to determine if it should actually log?

Reply:
The problem seems to be that this thing isn't finding `this.logger` and is therefor falling back to console. So, we need to either (a) figure out why this isn't picking up `this.logger` or (b) throw a little bit of logic into the fallback for when it can't find `this.logger`.

Reply:
Is this the only place were `this.logger` may not be found? Doesn't seem like we check it any where else we use it, but of course this is in a decorator

Reply:
yea, i'm pretty sure it is.